examples/persisted-documents/package.json

```diff
@@ -7,7 +7,7 @@
     "@graphql-yoga/plugin-persisted-operations": "1.7.2"
   },
   "devDependencies": {
-    "@graphql-typed-document-node/core": "3.1.2",
+    "@graphql-typed-document-node/core": "3.2.0-alpha-20230314161625-aafb8c1",
```

Comment:
would need to get a stable release out before we merge

Reply:
https://github.com/dotansimha/graphql-typed-document-node/pull/155 released it @beerose please update 😇